### PR TITLE
Add openqa-get-job-incidents

### PR DESCRIPTION
**Pull request by grisu48**
Adds file for querying the incidents of openQA test runs (OSD only).

### CODE REVIEW

File: openqa-get-job-incidents [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+if __name__ == "__main__" :
+	parser = argparse.ArgumentParser()
+	parser.add_argument("jobs", help="openQA jobs go here", nargs="+")
+	parser.add_argument("-d", "--diff", help="List only the diff between two jobs", action="store_true",default=False)
+	args = parser.parse_args()
+	
+	jobs = args.jobs
+	try :
+
```
Comment (Martchus) [inline openqa-get-job-incidents:87]:
```suggestion
```

Comment (grisu48) [reply openqa-get-job-incidents]: Done.

**Review by okurz — CHANGES_REQUESTED** [review]
Please change the file to use 4 tabs for indention. See existing files like https://github.com/os-autoinst/scripts/blob/master/openqa-trigger-bisect-jobs. Also please add at least a minimal corresponding test module in https://github.com/os-autoinst/scripts/tree/master/tests

**Comment by okurz** [comment]
please add at least a minimal corresponding test module in https://github.com/os-autoinst/scripts/tree/master/tests

**Review by okurz — CHANGES_REQUESTED** [review]
please add at least a minimal corresponding test module in https://github.com/os-autoinst/scripts/tree/master/tests

**Comment by grisu48** [comment]
No time right now to do additional work here. I might consider a resubmission later.

**Pull request closed without merging**